torch/jit/operator_upgraders.py

```diff
@@ -1,120 +0,0 @@
-"""
-This is the centralized file for all PyTorch operator upgraders.
-Each function definition here should satisfy following
-requirements:
-1. The body of the function must be Torchscriptable
-2. The naming convention of the upgraders should be:
-    <op_name>_<op_overload>_upgrader_<old_version>_<new_version>
-3. The name of the upgrader must be present in
-   torch/csrc/jit/operator_upgraders/version_map.h
-"""
-import torch
-from typing import List, no_type_check, Optional, Union
-
-# TODO (tugsuu) This context manager
-# forbids the tests to override torch.jit.script
-# Without it, torch.jit.load will fail due to
-# circular dependency
-with torch._jit_internal._disable_emit_hooks():
-    @torch.jit.script
-    def div_Tensor_0_3(self: torch.Tensor, other: torch.Tensor) -> torch.Tensor:
-        if (self.is_floating_point() or other.is_floating_point()):
-            return self.true_divide(other)
-        return self.divide(other, rounding_mode='trunc')
-
-    @torch.jit.script
-    def div_Scalar_0_3(self: torch.Tensor, other: Union[int, float]) -> torch.Tensor:
-        if (self.is_floating_point() or isinstance(other, float)):
-            return self.true_divide(other)
-        return self.divide(other, rounding_mode='trunc')
-
-    @torch.jit.script
-    def div_out_0_3(self: torch.Tensor, other: torch.Tensor, *, out: torch.Tensor) -> torch.Tensor:
-        if (self.is_floating_point() or other.is_floating_point() or out.is_floating_point()):
-            return self.true_divide(other, out=out)
-        return self.divide(other, rounding_mode='trunc', out=out)  # type: ignore[call-overload]
-
-    @torch.jit.script
-    def div__Tensor_0_3(self: torch.Tensor, other: torch.Tensor) -> torch.Tensor:
-        if (self.is_floating_point() or other.is_floating_point()):
-            return self.true_divide_(other)
-        return self.divide_(other, rounding_mode='trunc')
-
-    @torch.jit.script
-    def div__Scalar_0_3(self: torch.Tensor, other: Union[int, float]) -> torch.Tensor:
-        if (self.is_floating_point() or isinstance(other, float)):
-            return self.true_divide_(other)
-        return self.divide_(other, rounding_mode='trunc')
-
-    # TODO: since TS relies on typecheck comment of mypy
-    # adding type: ignore at specific lines in this function
-    # messes up our type refinement. For now, let's not check
-    # type here.
-    @no_type_check
-    @torch.jit.script
-    def full_0_4(size: List[int], fill_value: Union[int, float], *,
```

Comment:
Where are they moved to now?

Reply:
https://github.com/pytorch/pytorch/blob/master/torch/csrc/jit/operator_upgraders/upgraders_entry.cpp